### PR TITLE
feat(docs): update contributing guide

### PR DESCRIPTION
**Pull request by dionysuzx**
View the updated guide here: https://github.com/taikoxyz/taiko-mono/blob/9bcb5970ce6d49b60229a15dd3a848d49470b755/CONTRIBUTING.md

@odesium , if you could check the GitPOAP guide as well.

### CODE REVIEW

**Comment by d1onys1us** [comment]
closes #430

**Review by davidtaikocha — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,74 +1,93 @@
...
 
-Follow the [NatSpec format](https://docs.soliditylang.org/en/v0.8.16/natspec-format.html) for documenting smart contract source code. Please adhere to a few additional standards:
+# Claim a GitPOAP
 
-- Choose `/** */` over `///` for multi-line NatSpec comments, to save column space
-- Omit the usage of `@notice`, this will be automatically picked up so it will save column space and improve readability
-- Take advantage of inheritance for docs (such as documenting the interface), if you need to specify inherited docs use `@inheritdoc`
+We are rewarding community contributions with a GitPOAP. This guide explains the requirements to claim a GitPOAP.
+
+## XXXX Taiko Contributor GitPOAP
```
Comment (dantaik) [inline CONTRIBUTING.md:30]: Is this XXX a placeholder to be replaced soon?

**Review by dantaik — APPROVED** [review]

**Comment by odesium** [comment]
Instead of XXXX maybe 20XX

**Review by odesium — COMMENTED** [review]
Instead of XXXX maybe 20XX

**Pull request merged**